test/fixtures/ledger_transaction_fixtures.js

```diff
@@ -219,12 +241,47 @@ module.exports = {
     }
 
     return {
-      getPactified: () => {
-        return pactRegister.pactify(data)
-      },
-      getPlain: () => {
-        return data
+      getPactified: () => pactRegister.pactify(data),
+      getPlain: () => data
+    }
+  },
+  validTransactionEventsResponse: (opts = {}) => {
+    const data = {
+      transaction_id: opts.transaction_id || 'ht439nfg2l1e303k0dmifrn4fc',
+      events: (opts.payment_states) ? buildPaymentEvents(opts) : buildChargeEventStateWithDefaults(opts)
+    }
+
+    return {
+      getPactified: () => pactRegister.pactify(data),
+      getPlain: () => data
+    }
+  },
+  validTransactionSearchResponse: (opts = {}) => {
+    let results = []
+    opts.transactions.forEach(transaction => {
+      let result
+      transaction.gateway_account_id = opts.gateway_account_id
+      if (transaction.type === 'payment') {
+        transaction.includeCardDetails = true
+        transaction.includeRefundSummary = true
+        transaction.includeSettlementSummary = true
+        result = buildTransactionDetails(transaction)
+        results.push(result)
```

Comment:
current version doesn't increase readability so it might as well be:
```suggestion
        results.push(buildTransactionDetails(transaction))
```